test/martha/martha_v3.test.js

```diff
@@ -31,350 +48,6 @@ const mockResponse = () => {
-        contentType: 'application/json',
-        size: 6703858793,
-        timeCreated: '2018-06-27T14:28:06.398Z',
-        timeUpdated: '2018-06-27T14:28:06.398Z',
-        bucket: 'gdc-tcga-phs000178-controlled',
-        name: 'BRCA/RNA/RNA-Seq/UNC-LCCC/ILLUMINA/UNCID_2210188.c71ca9f7-248f-460c-b5d3-afb2c648fef2.110412_UNC13-SN749_0051_AB0168ABXX_4.tar.gz',
-        gsUri:
-            'gs://gdc-tcga-phs000178-controlled/BRCA/RNA/RNA-Seq/UNC-LCCC/ILLUMINA/UNCID_2210188.c71ca9f7-248f-460c-b5d3-afb2c648fef2.110412_UNC13-SN749_0051_AB0168ABXX_4.tar.gz',
-        googleServiceAccount: expectedGoogleServiceAccount,
-        hashes: {
-            md5: '2edd5fdb4f1deac4ef2bdf969de9f8ad'
-        }
-    };
-};
-
-const anvilDrsMarthaResult = (expectedGoogleServiceAccount) => {
-    return {
-        contentType: 'application/json',
-        size: 143562155,
-        timeCreated: '2020-07-08T22:52:53.194Z',
-        timeUpdated: '2020-07-08T22:52:53.194Z',
-        bucket: 'fc-secure-ff8156a3-ddf3-42e4-9211-0fd89da62108',
-        name: 'GTEx_Analysis_2017-06-05_v8_RNAseq_bigWig_files/GTEX-1GZHY-0011-R6a-SM-9OSWL.Aligned.sortedByCoord.out.patched.md.bigWig',
-        gsUri:
-            'gs://fc-secure-ff8156a3-ddf3-42e4-9211-0fd89da62108/GTEx_Analysis_2017-06-05_v8_RNAseq_bigWig_files/GTEX-1GZHY-0011-R6a-SM-9OSWL.Aligned.sortedByCoord.out.patched.md.bigWig',
-        googleServiceAccount: expectedGoogleServiceAccount,
-        hashes: {
-            md5: '18156430a5eea715b9b58fb53d0cef99'
-        }
-    };
-};
-
-const kidsFirstDrsMarthaResult = (expectedGoogleServiceAccount) => {
-    return {
-        contentType: 'application/json',
-        size: 55121736836,
-        timeCreated: '2018-05-23T16:32:32.594Z',
-        timeUpdated: '2018-05-23T16:32:32.594Z',
-        bucket: null, // expected, uses S3
-        name: null, // there is definitely a name in the server response, why isn't Martha using it?
-        gsUri: null, // expected, uses S3
-        googleServiceAccount: expectedGoogleServiceAccount,
-        hashes: {
-            etag: undefined
-        }
-    };
-};
-
-const hcaDosMarthaResult = (expectedGoogleServiceAccount) => {
-    return {
-        contentType: 'application/octet-stream',
-        size: 8933233597,
-        timeCreated: null,
-        timeUpdated: null,
-        bucket: "org-hca-dss-checkout-prod",
-        name: "blobs/44d320c9606e32d6df04d8a4023e6474efaa2f99de436e07f7241942972d5703.c3349268e528c844c528a427aa13034e72b7b39d.16c68f2306435b7cf990153b37adeb20-134.64d51664",
-        gsUri: "gs://org-hca-dss-checkout-prod/blobs/44d320c9606e32d6df04d8a4023e6474efaa2f99de436e07f7241942972d5703.c3349268e528c844c528a427aa13034e72b7b39d.16c68f2306435b7cf990153b37adeb20-134.64d51664",
-        googleServiceAccount: expectedGoogleServiceAccount,
-        hashes: {
-            sha256: '44d320c9606e32d6df04d8a4023e6474efaa2f99de436e07f7241942972d5703'
-        }
-    };
-};
-
 const dosObjectWithMissingFields = {
```

Comment:
Why remove all the other testing constants to `_martha_v3_resources.js` but not this and the next? I'm just curious. (Any of the non-shared ones, I could also argue could just be in the test function itself.)  I'm not trying to change your opinion.

Reply:
No reason, it's a lot shorter and my pattern-matcher missed it